tests/4-05-types.js

```diff
@@ -68,52 +68,54 @@ describe('Types', function() {
       });
       // Verifiable Credential MUST have a type specified
-      it('Verifiable Credential objects MUST have a type specified.',
-        async function() {
-          this.test.link = `https://w3c.github.io/vc-data-model/#types:~:text=the%20following%20table%20lists%20the%20objects%20that%20MUST%20have%20a%20type%20specified.`;
-          await assert.doesNotReject(endpoints.issue(require(
-            './input/credential-optional-type-ok.json')),
-          'Failed to accept a VC with additional type.');
-          await assert.rejects(endpoints.issue(require(
-            './input/credential-missing-required-type-fail.json')),
-          {name: 'HTTPError'},
-          'Failed to reject a VC missing the `VerifiableCredential` type.');
-        }
+      it(trimText(`Verifiable Credential objects MUST have a type 
+        specified including "VerifiableCredential".`),
+      async function() {
+        this.test.link = `https://w3c.github.io/vc-data-model/#types:~:text=the%20following%20table%20lists%20the%20objects%20that%20MUST%20have%20a%20type%20specified.`;
+        await assert.doesNotReject(endpoints.issue(require(
+          './input/credential-optional-type-ok.json')),
+        'Failed to accept a VC with additional type.');
+        await assert.rejects(endpoints.issue(require(
+          './input/credential-missing-required-type-fail.json')),
+        {name: 'HTTPError'},
+        'Failed to reject a VC missing the `VerifiableCredential` type.');
+      }
       );
       // Verifiable Presentation MUST have a type specified
-      it('Verifiable Presentation objects MUST have a type specified.',
-        async function() {
-          this.test.link = `https://w3c.github.io/vc-data-model/#types:~:text=the%20following%20table%20lists%20the%20objects%20that%20MUST%20have%20a%20type%20specified.`;
-          const presentationOptionalType = await endpoints.createVp({
-            presentation: require('./input/presentation-optional-type-ok.json')
-          });
-          await assert.doesNotReject(endpoints.verifyVp(
-            presentationOptionalType
-          ), 'Failed to accept VP with `@context` mapped type.');
-          await assert.rejects(
-            endpoints.verifyVp(require(
-              './input/presentation-missing-required-type-fail.json')),
-            {name: 'HTTPError'},
-            'Failed to reject VP missing `VerifiableCredential` type.');
-        }
+      it(trimText(`Verifiable Presentation objects MUST have a 
+        type specified including "VerifiablePresentation".`),
+      async function() {
+        this.test.link = `https://w3c.github.io/vc-data-model/#types:~:text=the%20following%20table%20lists%20the%20objects%20that%20MUST%20have%20a%20type%20specified.`;
+        const presentationOptionalType = await endpoints.createVp({
+          presentation: require('./input/presentation-optional-type-ok.json')
+        });
+        await assert.doesNotReject(endpoints.verifyVp(
```

Comment:
so I am actually -1 to the pattern used in the VCDM 2.0 suite as I think we should have an assertion that looks like at status code and also (in the future) looks at error objects. Also there is a reusable assertion in data-integrity-test-suite-assertion that could be used here `shouldBeErrorResponse` is the name I believe, but this suite does not use D.I. assertion so that is excusable.